app/templates/components/signatures/timeline-step.hbs

```diff
@@ -35,12 +35,43 @@
             <!-- dit verschilt voor verschillende organen -->
             {{#wu-button
                class="button u-spacer--top--tiny"
-               onClick=(action sign mockDocument.signedId)
+               onClick=(action (mut disabledOnderteken) false)
```

Comment:
The action should also be modified for the second signature button (= the button below the text 'Tweede handtekening vereist')